nautobot_ssot_aristacv/diffsync/models/nautobot.py

```diff
@@ -66,7 +69,7 @@ def create(cls, diffsync, ids, attrs):
             status=OrmStatus.objects.get(slug=attrs["status"]),
             device_type=device_type_object,
             device_role=role,
-            platform=OrmPlatform.objects.get(slug="arista_eos"),
+            platform=OrmPlatform.objects.get(slug=slugify(PLUGIN_SETTINGS["platform_name"])),
```

Comment:
I think this is going to break if the user ever uses something that's not Arista EOS. If you're trying to do a get with the slug we should actually specify `arista_eos` as the slug shouldn't be changing. We should always be using the netmiko OS name for the Platform slug IME.

Reply:
The use case I'm trying to address here is having an environment where there is already (prior to installation of this plugin) a Platform object for Arista EOS with a name that is not "arista.eos.eos". SInce there may be existing integrations with that I don't think we should force the user to change their existing platform object but rather make it possible for them to configure this plugin such that is adheres to their standard.

Reply:
Can you elaborate how exactly you see this breaking if the user chooses a different platform name?

Reply:
I guess the question really becomes, why does it have to be slug then? You're using the name and performing extra slugify action unnecessarily. Just do the get with the name in this case.

Reply:
@Kircheneer This needs to be reverted back, correct? I'm cutting a release as we have a customer waiting but let's circle back next week and get this merged.